iis-wf/iis-wf-referenceextraction/src/main/java/eu/dnetlib/iis/wf/referenceextraction/patent/PatentMetadataRetrieverJob.java

```diff
@@ -137,9 +137,16 @@ public static void main(String[] args) throws Exception {
             }
 
             // store final results
+            long entitiesReturnedFromCacheCount = entitiesReturnedFromCache.count();
+            long faultsReturnedFromCacheCount = inputJoinedWithCache
+                    .filter(x -> x._2._2.isPresent() && !x._2._2.get().isPresent())
+                    .count();
+            long processedEntitiesCount = retrievedPatentMeta.count();
+            long processedFaultsCount = faults.count();
```

Comment:
I guess we should track all the RDDs we are running `count()` method on. If they are also persisted in a later stage (e.g. `faults`) or are part of other operations which end up with an action (`retrievedPatentMeta` probably is the case) then they are potential subject for caching.